helm/app-operator/templates/rbac.yaml

```diff
@@ -199,7 +199,7 @@ metadata:
     {{- include "labels.common" . | nindent 4 }}
 rules:
   - apiGroups:
-      - extensions
```

Comment:
app-operator 1.1.x would never reconcile with k8s cluster under 1.16 so I think deleting this `extensions` would work.